example/windows/example/AutolinkedNativeModules.g.targets

```diff
@@ -3,7 +3,7 @@
   <!-- AutolinkedNativeModules.g.targets contents generated by "react-native autolink-windows" -->
   <ItemGroup>
     <!-- Projects from react-native-xaml -->
-    <ProjectReference Include="$(ProjectDir)..\..\..\package\windows\ReactNativeXaml\ReactNativeXaml.vcxproj">
+    <ProjectReference Include="$(ProjectDir)..\..\..\node_modules\react-native-xaml\windows\ReactNativeXaml\ReactNativeXaml.vcxproj">
       <Project>{0ff7027a-222c-4ffb-8f17-91d18bbaf7a8}</Project>
```

Comment:
same as with the SLN, this should keep autolinking the code in the repo, not the code from NPM